src/main/java/org/killbill/billing/plugin/vertex/VertexTaxCalculator.java

```diff
@@ -361,7 +373,12 @@ private SaleRequestType toTaxRequest(final Account account,
 
         SellerType sellerType = new SellerType();
         sellerType.setCompany(companyName);
-        sellerType.setDivision(companyDivision);
+
+        final String sellerDivision = PluginProperties.findPluginPropertyValue(SELLER_DIVISION, pluginProperties);
+        sellerType.setDivision(sellerDivision != null ? sellerDivision : defaultCompanyDivision);
+
+        sellerType.setAdministrativeOrigin(extractSellerAddress(pluginProperties));
```

Comment:
there should not be any default, we send what we get in property.

Reply:
removed default division logic usage